### ISSUE TITLE
Playback doesn't resume after being interrupted by alarm clock

### ISSUE THREAD

**Issue by knobhead** [y20k/transistor#132]
Hi, within the latest few versions of your wonderful Transistor app I've noticed the following behaviour:

1) I start playback manually (and doze away again)
2) At a given time my wake-up alarm rings and Transistor playback stops 
3) I tell my nasty alarm to shut the f*** up
4) Transistor's playback does not resume automatically. It's notification still shows the "stop" action button, as if it was stil playing. Tapping the "stop" button once changes it to the "start" button again. Tapping the notification's "start" button resumes playback correctly.

Is this intended behaviour? Somehow I feel that in previous versions, playback would resume automatically after I stopped the alarm. 

I'm running Transistor v2.2 on a Fairphone 2 with FP Open OS 17.04 (Android 5.1).

**Comment by y20k** [comment]
Hey @knobhead,

wow, dozing away can really help make open source software better 💤 😄. Thank you for pointing me to this bug. 

The bug in this case is: As soon as the playback stops, the user interface should reflect that change of state. This error should be fixed in the next release.

The main thing that bothers you was the intended and implemented behavior of the app. Transistor did not start playback after interruptions like phone calls or alarm clocks. But I checked: All audio apps I use do start playback again after a phone call.

So I looked up the documentation (see. [Responding to audio focus change](https://developer.android.com/guide/topics/media-apps/volume-and-earphones.html#audio-focus-change)): Transistor receives broadcasts when a so called `transient` event happens. 

I implemented a new behavior / the standard behavior (see [commit](https://github.com/y20k/transistor/commit/cf9c191c7061080e538b9842ba8aec5bb9f5b753)): As soon as Transistor receives the broadcast that a `transient` event is over, it starts playback again. I did test the behavior using the Phone app, the Clock app and Timely. It seems to work and the behavior "feels" right for a radio app.

The fix should be part of the next release, too.

THX again.
